src/app/modules/onboarding/module.nim

```diff
@@ -65,8 +65,8 @@ method delete*[T](self: Module[T]) =
   self.viewVariant.delete
   self.controller.delete
 
-method onAppLoaded*[T](self: Module[T]) =
-  self.view.appLoaded()
+method onAppLoaded*[T](self: Module[T], keyUid: string) =
+  self.view.appLoaded(keyUid)
```

Comment:
As discussed in DM, I don't quite like this. I guess we can leave as is, because it works, but I'll leave my opinion here.

It doesn't make sense for `appLoaded` to know the `keyUid`. This signal is about the app being loaded, it has nothing to do with "logged in" event. 

And actually, in the next line we destroy the whole onboarding module, right after emitting the signal.

Reply:
In onboarding store we have only two signals exposed: `appLoaded` and `accountLoginError`. For the UI it's useful to know for which uid the app was loaded (especially that it may be fresh, just created uid for new account). We may want to store some local metadata like settings or, like in that case, keychain. And we need to have the identifier for that.

From the UI side I think it's reasonable, if on the nim side can be done in better way, I'm open for propositions.